Dev dependancy issue with bytebuffer while implementing NextJS v13.4 App/ Directory

**Describe the bug**
Recently have been upgrading to NextJS's new App/ directory router with v13.4
While utilizing the Pages directory, this was not an issue. 

NextJS v13.4 (Page directory) (No Issue)
NextJS v13.1 (Page directory) (No Issue)
NextJS v13.4 (App directory) (Issue)
NextJS v13.1 (App directory) (Issue)

**Info:**

-   Suite Version: web
-   Browser: Chrome/Version 112.0.5615.137 (Official Build) (arm64)
-   OS: macOS Monterey, V12.5.1

**Reproduction**

I have recreated the issue in this bootstrap repo branch:
https://github.com/Seranged/EVM-FE-Bootstrap/tree/trezor-nextjs-issue-reproduction

1. yarn install
2. yarn dev
3. terminal says:

 warn ./node_modules/bytebuffer/dist/bytebuffer-node.js
Module not found: Can't resolve 'memcpy' in '/GitHub/EVM-bootstrap/node_modules/bytebuffer/dist'

Import trace for requested module:
./node_modules/bytebuffer/dist/bytebuffer-node.js
./node_modules/@trezor/transport/lib/lowlevel/receive.js
./node_modules/@trezor/transport/lib/bridge/v2.js
./node_modules/@trezor/transport/lib/index.js
./node_modules/@trezor/connect/lib/data/config.js
./node_modules/@trezor/connect-web/lib/index.js
./src/app/layout.tsx

**Expected behavior**
There shouldn't be a dev dependency issue, I think it's to do with how the Trezor dev dependencies are working in combination with the new NextJS App directory, as the exact same repo but utilizing the Page/ directory works perfectly fine.

**Screenshots**
<img width="1258" alt="image" src="https://user-images.githubusercontent.com/80223622/236628800-a6f34de8-e46c-46f1-8282-ddb3fd7dbe6d.png">

--- a/packages/connect/src/data/config.ts
+++ b/packages/connect/src/data/config.ts
@@ -1,6 +1,6 @@
 // origin: https://github.com/trezor/connect/blob/develop/src/data/config.json
 
-import { TREZOR_DESCS } from '@trezor/transport';
+import { TREZOR_DESCS } from '@trezor/transport/lib/constants';
 
 export const config = {
     webusb: TREZOR_DESCS,

